buildspec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+
+phases:
+  install:
+    commands:
+      - sudo apt-get update
+      - sudo apt-get install apt-transport-https
+      - curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+      - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+      - sudo apt-get update && sudo apt-get install yarn
+  pre_build:
+    commands:
+      - yarn install
+  build:
+    commands:
+      - yarn build:all
+      - ./node_modules/mocha/bin/mocha -r ts-node/register test/**/*.spec.ts
+artifacts:
+  files:
+    - 'handler.js'
+  discard-paths: yes
+  base-directory: src/handlers
+cache:
+  paths:
+    - '/root/.npm/**/*'
```

Comment:
It also seems to be saving the entirety of the `/node_modules/` directory as an artifact, according to the logs.  This _probably_ isn't necessary if we move that to cache, and will help make sure build times are quick.

<img width="1218" alt="screen shot 2018-02-21 at 1 01 26 pm" src="https://user-images.githubusercontent.com/3791752/36505177-8d034af4-1707-11e8-9953-10650a570ab9.png">

Reply:
Hmm, that isn't reflected in that image. But I can add it to the `exclude` section for artifacts just to be sure.

Reply:
I bet this is for the CD PR, actually.  The command `zip -r build.zip *` is running, so I'm betting that if we remove it (per the comments on the other PR) then that should fix this one.